server/handler_test.py

```diff
@@ -63,8 +63,8 @@ def test_content_correctness(self):
         # was generated by out Java client. This test confirms
         # That the content pulled down for a particular day is
         # the same as that pulled down by earlier invocations from Java
-        user = os.environ.get("ST_UN")
-        password = os.environ.get("ST_PW")
+        user = os.environ.get("SPACETRACK_USERNAME")
+        password = os.environ.get("SPACETRACK_PASSWORD")
```

Comment:
I have updated the .env in a couple places to duplicate both with a note that `ST_*` is deprecated :)